### ISSUE TITLE
Manifest still requires `activationEvents`, but shouldn't anymore

### ISSUE THREAD

**Issue by alefragnani** [microsoft/vscode-vsce#808]
The VS Code 1.74 release added [Implicit Activation Events](https://code.visualstudio.com/updates/v1_74#_implicit-activation-events-for-declared-extension-contributions), but `vsce` doesn't seems to have been updated to support that yet.

I just tried to publish an update for one of my extensions, which now doesn't have any `activationEvents` anymore, and even `vsce ls` doesn't work

```shell
 ERROR  Manifest needs the 'activationEvents' property, given it has a 'main' property.
```

BTW, `vsce` is updated to `2.16`, as described https://code.visualstudio.com/updates/v1_74#_renaming-of-vsce-to-vscodevsce

Thank you

**Comment by alefragnani** [comment]
As a workaround, an empty `activationEvents` seems to bypass the `vsce` validation. 

But I wonder if the Implicit Activation Events feature will still work as expected, or will _intentionally_ leave it empty. In my case, were `commands` and it worked perfectly.

**Comment by sandy081** [comment]
@joaomoreno It seems there is no new vsce package with this fix to verify.

**Comment by joaomoreno** [comment]
Will work on getting a pre-release pipeline: https://github.com/microsoft/vscode-vsce/issues/817

For the time being I will simply release a new version: https://github.com/microsoft/vscode-vsce/pull/816

**Comment by noppej** [comment]
@joaomoreno I can still reproduce this problem against 
```
% vsce --version
2.17.0
```

**Comment by PierreBelmondo** [comment]
Try with vscode engine 1.74.0 or superior:

```
  "engines": {
    "vscode": "^1.74.0"
  },
```

**Comment by ajyoon** [comment]
Is this really fixed? I'm still getting this error on vsce 2.15.0. My package.json has

```json
  "engines": {
    "vscode": "^1.75.0"
  },
```

**Comment by default-writer** [comment]
> Is this really fixed? I'm still getting this error on vsce 2.15.0. My package.json has
> 
> ```json
>   "engines": {
>     "vscode": "^1.75.0"
>   },
> ```

```js
  "icon": "icon.png",
  "activationEvents": [
    "onStartupFinished"
  ],
```

**Comment by alefragnani** [comment]
@ajyoon ,

I'm using vsce 2.18.0 and it works just fine. 

Just published a new release of my https://github.com/alefragnani/vscode-copy-word extension a few days ago, and it has no `activationEvents` in its `package.json` file. The `engines` is defined as `^1.74.0` btw.

Worth updating your vsce to be sure.

Hope this helps
